index.html

```diff
@@ -5,8 +5,8 @@
     <meta charset="UTF-8">
     <title>Document</title>
     <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+    <script type="text/javascript" src="todo.js"></script>
```

Comment:
ideally you should add script tag at the end of the html document just before closing the </body> tag.